doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -38,38 +38,40 @@ more, provisioning a DB device makes more sense.  The BlueStore
 journal will always be placed on the fastest device available, so
 using a DB device will provide the same benefit that the WAL device
 would while *also* allowing additional metadata to be stored there (if
-it will fit).
+it will fit).  This means that if a DB device is specified but an explicit
+WAL device is not, the WAL will be implicitly colocated with the DB on the faster
+device.
 
-A single-device BlueStore OSD can be provisioned with::
+A single-device (colocated) BlueStore OSD can be provisioned with::
 
   ceph-volume lvm prepare --bluestore --data <device>
 
 To specify a WAL device and/or DB device, ::
 
   ceph-volume lvm prepare --bluestore --data <device> --block.wal <wal-device> --block.db <db-device>
 
-.. note:: --data can be a Logical Volume using the vg/lv notation. Other
-          devices can be existing logical volumes or GPT partitions
+.. note:: ``--data`` can be a Logical Volume using  *vg/lv* notation. Other
+          devices can be existing logical volumes or GPT partitions.
 
 Provisioning strategies
 -----------------------
-Although there are multiple ways to deploy a Bluestore OSD (unlike Filestore
-which had 1) here are two common use cases that should help clarify the
-initial deployment strategy:
+Although there are multiple ways to deploy a BlueStore OSD (unlike Filestore
+which had just one), there are two common arrangements that should help clarify
+the deployment strategy:
 
 .. _bluestore-single-type-device-config:
 
 **block (data) only**
 ^^^^^^^^^^^^^^^^^^^^^
-If all the devices are the same type, for example all are spinning drives, and
-there are no fast devices to combine these, it makes sense to just deploy with
-block only and not try to separate ``block.db`` or ``block.wal``. The
-:ref:`ceph-volume-lvm` call for a single ``/dev/sda`` device would look like::
+If all devices are the same type, for example all are rotational drives, and
+there are no fast devices to exploit for metadata, it makes sense to specifiy the
```

Comment:
to use for metadata

Reply:
Agreed, I struggled with this a bit.  `use` is less lurid ;)